docs/kinesis/kinesis_connector.md

```diff
@@ -0,0 +1,103 @@
+# Exasol Kinesis Connector 
+
+Exasol Kinesis Connector provides UDF scripts which allow users import data
+from a Kinesis Stream to an Exasol table.
+
+## Prerequisites
+
+* A running Exasol cluster with a version 6.0 or later.
+* An AWS account with all necessary permissions to read from a Kinesis Stream.
+* An AWS Kinesis producer which sends the data to a stream in the valid JSON
+ format. 
+ An example of a valid JSON string: 
+ `{"sensorId": 17,"currentTemperature": 147,"status": "WARN"}`.
+ An example of an invalid JSON format:
+ `{"sensorId": 17,"currentTemperature": 147,"status": "WARN",}`. The Exasol
+ Kinesis Connector will not parse this string correctly.
+
+## Deployment
+
+Please refer to the [Deployment Guide](../deployment_guide.md) till the section
+`Upload the JAR file to the bucket` (inclusive) to prepare a bucket and a JAR.
+
+## Preparing a table for data
+
+To store the data from a stream we need a table. 
+The columns in the Exasol table have to imitate the data types of the data
+ stored in the stream and also be in the exact same order.
+ 
+The table also requires two additional columns to store the Kinesis metadata:  
+    `kinesis_shard_id VARCHAR(2000), 
+     kinesis_sequence_number VARCHAR(2000)`
+These two columns must be in the end of the table.
+ 
+For example, we have a stream with the following data: 
+ `{"sensorId": 17,"currentTemperature": 147,"status": "WARN"}`
+We create a table for this stream:
+
+```sql
+OPEN SCHEMA <schema_name>;
+
+CREATE OR REPLACE TABLE <schema_name>.<table_name> 
+    (sensorId DECIMAL(18,0), 
+    currentTemperature DECIMAL(18,0), 
+    status VARCHAR(100), 
+-- Required for importing data from Kinesis
+    kinesis_shard_id VARCHAR(2000), 
+    kinesis_sequence_number VARCHAR(2000));
+
```

Comment:
To be consistent with the Kafka, could you please move these two extra columns as first two columns of table? It is helpfule to make them visible immediately as additional required columns.

Reply:
We decided not to do it